packages/apollo-server-cloudflare/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "apollo-server-cloudflare",
   "version": "2.9.9",
-  "description": "Production-ready Node.js GraphQL server for Cloudflare workers",
+  "description": "Production-ready GraphQL server for Cloudflare workers",
```

Comment:
since workers leverage v8 rather than the nodejs runtime it did not feel right to call this a nodejs server